Lesson5/Task6.cs

```diff
@@ -0,0 +1,51 @@
+namespace CSharpFundamentals.Lesson5;
+
+public class Task6
+{
+    public void StringTests()
+    {
+        try
+        {
+            Console.Write("Enter string: ");
+            string str1 = Console.ReadLine();
+
+            Console.Write("Enter string: ");
+            string str2 = Console.ReadLine();
+
+            if (String.IsNullOrEmpty(str1) || String.IsNullOrEmpty(str2))
+            {
+                throw new Exception("String is null or empty!");
+            }
+
+            Print(str1, str2);
+
+            Console.WriteLine();
+
+            bool compare = ComparingStrings(str1, str2);
```

Comment:
Method names must be like imperative sentence: not ComparingStrings -> CompareStrings